### ISSUE TITLE
Inconsistency in casing of "iat" claim between descriptor and header ToString methods

### ISSUE THREAD

**Issue by 0x15e** [ycrumeyrolle/Jwt#429]
This is just a tiny inconsistency I noticed in some testing. It probably wouldn't affect anyone unless they're abusing the `ToString` methods of `JwsDescriptor` and `Jwt`.

The following code:
```
var descriptor = new JwsDescriptor
{
    Algorithm = SignatureAlgorithm.None,
    IssuedAt = DateTime.UtcNow,
    ExpirationTime = DateTime.UtcNow.AddHours(1),
    Issuer = "https://idp.example.com/",
    Audience = "636C69656E745F6964"
};

Console.WriteLine("Descriptor ToString():");
Console.WriteLine(descriptor.ToString());
Console.WriteLine();

var tokenBytes = new JwtWriter().WriteToken(descriptor);

Console.WriteLine("Jwt ToString():");
Console.WriteLine(new JwtReader().TryReadToken(tokenBytes, TokenValidationPolicy.NoValidation).Token?.ToString());
```

Produces the following output:
```
Descriptor ToString():
{
  "alg": "none"
}
.
{
  "iat": 1588358068,
  "exp": 1588361668,
  "iss": "https://idp.example.com/",
  "aud": "636C69656E745F6964"
}

Jwt ToString():
{
  "alg": "none"
}.{
  "aud": "636C69656E745F6964",
  "iss": "https://idp.example.com/",
  "exp": 1588361668,
  "Iat": 1588358068
}
```

Note that the descriptor `ToString` output correctly has the "iat" claim in lowercase but the Jwt `ToString` output has it with a capital "I" ("Iat").

[This line](https://github.com/ycrumeyrolle/Jwt/blob/1bce9d4efff47ce9e73c602caed8a1e9bf57293b/src/JsonWebToken/JwtPayload.cs#L29) seems to be the source of the problem.

**Comment by ycrumeyrolle** [comment]
As you mentioned, the `ToString` should not be the prefered way to use a JWT except for debugging on writing to the console. 
Good catch!

**Comment by ycrumeyrolle** [comment]
Fixed in [v1.3.0](https://www.nuget.org/packages/JsonWebToken/)
